.github/workflows/test_build_deploy.yml

```diff
@@ -130,7 +130,7 @@ jobs:
 
       # Stage 1: https://wire-webapp-edge.zinfra.io/
       - name: Deploy edge build to Elastic Beanstalk
-        if: env.BRANCH_NAME == 'dev'
+        if: env.BRANCH_NAME == 'edge'
```

Comment:
This was a change we did a little while ago for MLS testing. But this should not stay there anymore (this would override the `edge` env with whatever is pushed to `dev`)